t/Cryptopcurrency.t

```diff
@@ -128,7 +128,7 @@ ddg_spice_test(
     # Things that should probably work but it doesn't at the moment.
     'ppc ftc 400' => undef,
     '499 nmc = ? usd' => undef,
-    
+    '500 etherium in btc' => undef, 
```

Comment:
We should do as @tagawa said and update this test to be a valid query.

Also add tests for any new triggers you have introduced.